Dockerfile.test.cpu

```diff
@@ -27,6 +27,13 @@ RUN apt-get install -y --no-install-recommends \
         build-essential \
         g++-4.8
 
+# install g++-7
+RUN apt-get install -y --no-install-recommends software-properties-common
+RUN apt-get update
```

Comment:
I don't think this line is necessary.

Reply:
These lines install the `add-apt-repository` command. Otherwise, we cannot add the test repo.